packages/esm-appointments-app/src/homepage-tile/appointments-tile.scss

```diff
@@ -0,0 +1,42 @@
+@use '@carbon/type';
+@use '@carbon/colors';
+@import '~@openmrs/esm-styleguide/src/vars';
+
+.tileContent {
```

Comment:
Not a show stopper but the contents of this file are being duplicated in the esm-visits as well. Can we make this reusable?

Reply:
Is there a way to share stylesheets across different esms?

Reply:
The only current way is via the style guide but let's leave this for now.